src/style.css

```diff
@@ -1,3 +1,70 @@
 body {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, 'Helvetica Neue', Arial, sans-serif;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.logo {
+  width: 40px;
+  height: 40px;
+}
+
+.menu {
+  width: 100%;
+  display: flex;
+  list-style: none;
+  justify-content: flex-end;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 60px;
+  color: #000;
+  margin: 0 10px;
+  padding: 0;
+  text-decoration: none;
+  text-align: center;
+  display: inline-block;
```

Comment:
Be consistent with your vertical margins. You have a vertical margin on `.nav__link` (line 35), which can lead to margin collapse issues. Decide on a single direction for vertical margins, either top or bottom, to maintain consistency and predictability in your layout.